components/ModSubmissionListContainer.vue

```diff
@@ -16,27 +16,60 @@
             {{ $t("modPanelSubmissionList.submitted") }}
         </div>
 
-        <div v-for="(fakeDatum, index) in modScreenStore.fakeData" :key="index" class="grid grid-cols-subgrid col-span-5 bg-gray-200" >
-            <button @click="handleClickToSubmissionForm(fakeDatum.id)"
-            :data-testid='`mod-submission-list-item-${index + 1}`'
+        <div v-if="hasSubmissions" v-for="(submission, index) in modSubmissionsStore.submissionsData" :key="index"
             class="grid grid-cols-subgrid col-span-5 bg-gray-200">
+            <button @click="handleClickToSubmissionForm(submission.id)"
+                :data-testid='`mod-submission-list-item-${index + 1}`'
+                class="grid grid-cols-subgrid col-span-5 bg-gray-200">
                 <span class="text-start">{{ index + 1 }}</span>
-                <span class="text-start">{{ $t(fakeDatum.name) }}</span>
-                <span class="text-start">{{ $t(fakeDatum.status) }}</span>
-                <span class="text-start">{{ $t(fakeDatum.modified) }}</span>
-                <span class="text-start">{{ $t(fakeDatum.submitted) }}</span>
+                <span class="text-start">{{ submission.healthcareProfessionalName }}</span>
+                <span class="text-start">{{ getSubmissionStatus(submission) }}</span>
+                <span class="text-start">{{ convertDateToUTC(submission.updatedDate) }}</span>
+                <span class="text-start">{{ convertDateToUTC(submission.createdDate) }}</span>
             </button>
         </div>
+        <div v-else>
+            {{ $t("modPanelSubmissionList.noSubmissions") }}
+        </div>
+
     </div>
 </template>
 
 
 
 <script setup lang="ts">
-import { computed } from 'vue'
+import { computed, onMounted } from 'vue'
 import { useModerationScreenStore } from '~/stores/moderationScreenStore';
+import { useModerationSubmissionsStore } from '~/stores/moderationSubmissionsStore';
+import { Submission } from '~/typedefs/gqlTypes';
+import { useNuxtApp } from "#app";
+const { $i18n } = useNuxtApp();
 
 const modScreenStore = useModerationScreenStore()
+const modSubmissionsStore = useModerationSubmissionsStore()
+
+onMounted(async () => {
+    await modSubmissionsStore.getSubmissions()
+})
+
+const hasSubmissions = computed(() => modSubmissionsStore.submissionsData.length > 0)
+
+const getSubmissionStatus = (submission: Submission) => {
+    if (submission.isUnderReview) {
+        return $i18n.t("modPanelSubmissionList.underReview")
+    }
+    if (submission.isApproved) {
+        return $i18n.t("modPanelSubmissionList.approved")
+    }
+    if (submission.isRejected) {
+        return $i18n.t("modPanelSubmissionList.rejected")
+    }
+    return $i18n.t("modPanelSubmissionList.new")
+}
+
+const convertDateToUTC = (isoString: string) => {
```

Comment:
this is a locale string, not UTC, correct? 
maybe I would call this something like `convertDateToLocalTime`